doc/source/serve/key-concepts.md

```diff
@@ -26,8 +29,10 @@ class MyFirstDeployment:
       return self.msg
 
 my_first_deployment = MyFirstDeployment.bind("Hello world!")
-handle = serve.run(my_first_deployment)
-print(ray.get(handle.remote())) # "Hello world!"
+handle: DeploymentHandle = serve.run(my_first_deployment).options(
+    use_new_handle_api=True,
+)
+print(handle.remote().result()) # "Hello world!"
 ```
```

Comment:
Should this be moved to a .py file under `doc_code` so it can be tested?

Reply:
yea it should, I can do that